docs/manuals/run-deploy/deploy/databricks/index.md

```diff
@@ -0,0 +1,124 @@
+# Deploy on Databricks
+
+Databricks is a standard platform for data scientists and big data engineers. It can provide a web-based laptop connected to your data source within your cloud provider. Nevertheless, Databricks does not provide a direct path to developing a web application. This documentation will show you how to adapt your infrastructure so that you can create a Taipy application in your Databricks.
+
+!!! Note
+
+    Taipy is currently only available with Databricks **Standard** runtimes.
+
+!!! Warning
+
+    We recommend using Databricks deployments for testing or demonstration purposes only.
+
+In the current section, we consider the following as prerequisites:
+
+- Knowledge of Databricks.
+- Knowledge of SSH.
+- A Databricks cluster in operation.
+- A Linux-based machine that can communicate with your local machine and Databricks.
+- [:material-arrow-right: Running a Taipy application](../../run/index.md)
+
+!!! Note
+
+    If you don't have a Linux-based machine, you can use a cloud provider as Azure to
+    [create one](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal?tabs=ubuntu).
+
+## Running your application on Databricks
+
+The first step consists in running your Taipy application on Databricks.
+Let's consider the following application:
+
+```python linenums="1"
+{%
+include-markdown "../../code_sample/basic_gui_rest_app.py"
+comments=false
+%}
+```
+
+Connect to Databricks and run the previous code in a Databricks Notebook. You should obtain a similar output:
```

Comment:
"You should obtain a similar output"
=>
"You should obtain an output similar to the following"